src/editing_tools/AdjustTool.vala

```diff
@@ -146,18 +160,26 @@ public class EditingTools.AdjustTool : EditingTool {
     }
 
     private abstract class AdjustToolCommand : Command {
-        protected weak AdjustTool owner;
+        public weak AdjustTool owner { get; construct; }
+        public string name { get; construct; }
+        public string explanation { get; construct; }
```

Comment:
Watch out for this. `name` and `explanation` are both properties of `Command`, so we don't want to override them here

Reply:
Thanks! I'll change it right away